### PR TITLE
ci: add hadolint and flake8 linters

### PR DESCRIPTION
**Pull request by audrium**
addresses https://github.com/reanahub/reana/issues/450

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,18 @@
 # This file is part of REANA.
-# Copyright (C) 2017, 2018 CERN.
+# Copyright (C) 2017, 2018, 2020 CERN.
```
Comment (mvidalgarcia) [inline Dockerfile:2]: I guess also 2019?

Would it be possible to keep a range on these cases? e.g. `Copyright (C) 2017-2020 CERN. cc/ @tiborsimko @diegodelemos

**Review by mvidalgarcia — APPROVED** [review]
Works well on my end ✅

File: Dockerfile (hunk 1 above)
Comment (audrium) [reply Dockerfile:2]: Guess this is similar case to [this one](https://github.com/reanahub/reana-workflow-monitor/pull/39#discussion_r525906051), that's why I haven't added 2019

Comment (mvidalgarcia) [reply Dockerfile:2]: I see.. anyway not a big deal! 

:shipit:

**Pull request merged**